sklearn/multioutput.py

```diff
@@ -365,3 +368,232 @@ def score(self, X, y):
                              format(n_outputs_, y.shape[1]))
         y_pred = self.predict(X)
         return np.mean(np.all(y == y_pred, axis=1))
+
+
+class ClassifierChain(BaseEstimator):
+    """A multi-label model that arranges binary classifiers into a chain.
+
+    Each model makes a prediction in the order specified by the chain using
+    all of the available features provided to the model plus the predictions
+    of models that are earlier in the chain.
+
+    Parameters
+    ----------
+    base_estimator : estimator
```

Comment:
Possibly. We'd have to think about how this would work with random chain orders but I could imagine this being a useful feature.